vm/src/obj/objint.rs

```diff
@@ -233,6 +233,12 @@ impl PyInt {
     {
         if cls.is(&vm.ctx.types.int_type) {
             Ok(vm.ctx.new_int(value).downcast().unwrap())
+        } else if cls.is(&vm.ctx.types.bool_type) {
+            if PyInt::from(value).value.is_zero() {
```

Comment:
You can use `ctx.new_bool` here, and you don't need the `PyInt::from(value). value`, that's the same thing as `value`